### DISCUSSION TITLE
Using Other Provided Environments

### DISCUSSION THREAD

**Discussion by sarah-seitanakis** [microsoft/AirSim#3346] [Support Q&A] [answered]
Hello, I am trying to use the other pre-made environments (like the ones shown in the AirSim demo videos). I found what I think is the right place to download the zips: [https://github.com/microsoft/AirSim/releases/tag/v1.4.0-windows](url). 

However, I am a bit confused because the files from unzipping are not set up like the example 'Blocks' environment from the repo. Maybe I have missed something. 

The documentation for using an environment is written for 'Blocks' (it was very helpful for 'Blocks' and I was able to get that working just fine) but I cannot find the equivalent files in the folder that was unzipped from the released environments.

It would be great if I could get a little bit of help using the other environments! Thank you :)

**Comment by ahmed-elsaharti** [comment, marked as answer]
Hi @sarah-seitanakis , the environments provided in the [releases section](https://github.com/microsoft/AirSim/releases/tag/v1.4.0-windows) you're referring to are pre-compiled binaries (ie: final executable files) which is why they don't contain the .sln and .uproject files like the blocks environment in /Unreal/Environments/Blocks. In other words, you can't open them as an UnrealEngine project to modify them.

**Reply by sarah-seitanakis** [reply]
Thank you so much for your response, @ahmed-elsaharti. Are the any instructions for how to run a simulation with the pre-compiled binaries?

**Reply by ahmed-elsaharti** [reply]
@sarah-seitanakis , Sorry for replying late, [Here's the docs page](https://microsoft.github.io/AirSim/use_precompiled/) outlining how to run the binaries. In general, you would just need to run the .exe executable (for the Windows binaries) under `WindowsNoEditor` after unzipping the zip file you downloaded:
1. ![image](https://user-images.githubusercontent.com/49626509/106058244-9a103580-60b6-11eb-9500-55f632031ca8.png)
2. ![image](https://user-images.githubusercontent.com/49626509/106058311-b0b68c80-60b6-11eb-8e20-988fcf82627d.png)
3. ![image](https://user-images.githubusercontent.com/49626509/106058339-b90ec780-60b6-11eb-9204-844dd3488766.png)
4. ![image](https://user-images.githubusercontent.com/49626509/106058352-be6c1200-60b6-11eb-8108-e0f204a5e927.png)
5. ![image](https://user-images.githubusercontent.com/49626509/106058378-c75ce380-60b6-11eb-9498-c2c0b1082798.png)
This launches the simulation directly:
![image](https://user-images.githubusercontent.com/49626509/106058617-1c98f500-60b7-11eb-9b6e-05154fe561e0.png)

**Reply by sarah-seitanakis** [reply]
Thank you so much! I really appreciate all of your help, @ahmed-elsaharti.
